build.gradle

```diff
@@ -56,6 +57,8 @@ ext.libraries = [
                 "junit:junit:${versions.junit4}",
                 "junit:junit-dep:${versions.junit4}",
         ],
+        hadoopCommon   : "org.apache.hadoop:hadoop-common:2.7.7",
```

Comment:
nit: refactor version to a const in `versions`.

Reply:
done